leftwm-core/src/display_servers/xlib_display_server/xwrap/getters.rs

```diff
@@ -309,6 +309,30 @@ impl XWrap {
         Ok(attrs)
     }
 
+    /// Returns a windows class `WM_CLASS`
+    // `XGetClassHint`: https://tronche.com/gui/x/xlib/ICC/client-to-window-manager/XGetClassHint.html
+    #[must_use]
+    pub fn get_window_class(&self, window: xlib::Window) -> Option<(String, String)> {
+        unsafe {
```

Comment:
Not super sure from a xlib point of view, but would it make sense to have `XAtom.WMClass: XClassHint` ([here](https://github.com/leftwm/leftwm/blob/main/leftwm-core/src/display_servers/xlib_display_server/xatom.rs#L14)) and move this logic to its [new constructor](https://github.com/leftwm/leftwm/blob/main/leftwm-core/src/display_servers/xlib_display_server/xatom.rs#L267)?

Reply:
The XClassHint isn't an atom but is of type strut with this definition:
```c
typedef struct {
	char *res_name;
	char *res_class;
} XClassHint;
```

Reply:
Yes, that's why I wasn't sure on the xlib side; it may be strange that `lefwm::XAtom` contains a `xlib::XClassHint`. What you wrote seems fine, then.